docs/en/latest/building-apisix.md

```diff
@@ -45,16 +45,10 @@ To build and package APISIX for a specific platform, see [apisix-build-tools](ht
 
 ## Building APISIX from source
 
-First of all, we need to specify the version `APISIX_VERSION` to be installed:
-
-```shell
-APISIX_VERSION='3.9.0'
-```
-
 Then, you can run the following command to clone the APISIX source code from Github:
 
 ```shell
-git clone --depth 1 --branch ${APISIX_VERSION} https://github.com/apache/apisix.git apisix-${APISIX_VERSION}
+git clone --depth 1 --branch master https://github.com/apache/apisix.git
```

Comment:
@shreemaan-abhishek are these the agreed upon changes?

![image](https://github.com/apache/apisix/assets/39619599/0479f1c9-834a-4ad1-9b0a-dd2de979e160)

I dont have additional comments otherwise

Reply:
yes.